cmd/curve.go

```diff
@@ -29,7 +30,11 @@ var curveCmd = &cobra.Command{
 		for _, config := range configuration.CurrentConfig.Fans {
 			if config.HwMon != nil {
 				for _, controller := range controllers {
-					if controller.Platform == config.HwMon.Platform {
+					matched, err := regexp.MatchString(config.HwMon.Platform, controller.Platform)
```

Comment:
If we change the behaviour again anyway, can we introduced a "case-insensitive" search for all of the comparisons too?
I had problems in the past with that too.

Reply:
Made the search case insenitive